app/views/catalog/_st_angular_form.html.haml

```diff
@@ -1,4 +1,5 @@
 - @angular_form = true
+= react('AnsiblePlayBookEditCatalogForm', {:initialData => {:catalogFormId => params[:id], :available_catalogs => @available_catalogs, :zones => @zones}})
```

Comment:
The prop`:available_catalogs` can be used like `:availableCatalogs` this to avoid lint errors if any in component

Reply:
made the changes . Thanks